spikingjelly/activation_based/ann2snn/examples/cnn_mnist.py

```
@@ -36,16 +66,219 @@ def val(net, device, data_loader, T=None):
+        ann2snn.download_url(checkpoint_url, str(checkpoint_path))
+    except KeyError as exc:
+        if exc.args != ("content-length",):
+            raise
+        headers = {
+            "User-Agent": (
+                "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:67.0) "
+                "Gecko/20100101 Firefox/67.0"
+            )
+        }
+        tmp_path = checkpoint_path.with_suffix(checkpoint_path.suffix + ".tmp")
+        response = requests.get(checkpoint_url, headers=headers, stream=True)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)